java/arcs/core/analysis/InformationFlowLabels.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.analysis
+
+import java.util.BitSet
```

Comment:
Unfortunately this won't work outside of the jvm. That's ok for now, we can come back and fix this later when that becomes an issue.

Reply:
Should we track this in a bug?